src/python/foglamp/core/scheduler/scheduler.py

```diff
@@ -662,17 +662,17 @@ def _schedule_first_task(self, schedule, current_time):
             },
             "max_completed_task_age_days": {
                 "description": "The maximum age, in days (based on the start time), for a rows "
-                               "in the tasks table that do not have a status of 'running'",
+                               "in the tasks table that do not have a status of running",
```

Comment:
@MarkRiddoch As @ashwinscale has pointed out notice the word 'running' in single quotes that will produce an error if it goes through storageclient-> storage service
```
{'message': 'ERROR:  syntax error at or near "running"\nLINE 1: ... in the tasks table that do not have a status of \'running\'",...\n                                                             ^\n', 'entryPoint': 'insert', 'retryable': False}
```

So I removed these quotes and now it is working fine. Also I tried with `\'running\'` and yes it also works with latest storage build (including FOGL-690 changes)

I have added one unit test regarding it and skipped with `FOGL-690` (tests.test_configuration_manager.TestConfigurationManager#test_create_category_with_quoted_json_data). So I thought that fix would be in FOGL-690 itself. 

Kindly advise!